features/upgrade/etcd/upgrade.feature

```diff
@@ -38,7 +38,7 @@ Feature: basic verification for upgrade testing
   @4.11 @4.10 @4.9 @4.8 @4.7 @4.6
   @vsphere-ipi @openstack-ipi @gcp-ipi @baremetal-ipi @azure-ipi @aws-ipi
   @vsphere-upi @openstack-upi @gcp-upi @baremetal-upi @azure-upi @aws-upi
-  @proxy @noproxy @disconnected @connected
+  @proxy @noproxy @connected
```

Comment:
You need to rebase your PR and remove the same tag from prepare step, too.

Reply:
done. 
@liangxia ,
Could you please have a look.